web/src/styles/app.css

```
@@ -8665,3 +8657,71 @@ button.org-person-name {
 .agent-join-light--done { background: var(--t-panel2); }
 .agent-join-light--pending { opacity: 0.6; border-style: dashed; }
 .agent-join-light--active { box-shadow: 3px 3px 0 var(--d-shadow); }
+
+/* #1044 频道头部重做：头像堆叠（人圆 / agent 方，状态环 = 可达性）+ 人话模式芯片。 */
+.presence-stack { display: inline-flex; align-items: center; margin-right: 2px; }
+.presence-ava {
+  --ah: 200;
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)